### ISSUE TITLE
Cannot find name 'Ng2Dropdown' (again)

### ISSUE THREAD

**Issue by mtfurlan** [Gbuomprisco/ng2-tag-input#221]
So in [issue 110](https://github.com/Gbuomprisco/ng2-tag-input/issues/110#issuecomment-281945599) it was mentioned that going from version `0.8.6` to  `0.8.6` should fix this issue of not being able to find `Ng2Dropdown` from `ng2-material-dropdown`

That worked for me three days ago.

Today, a different team member started work on the project, and they had the issue again.
I thought this was odd, so I `rm -rf node_modules && npm i`
Now I have the issue again.

We're running a webpack based project, but I suspect it has something to do with the seed we're based on.

```
[at-loader] node_modules/ng2-tag-input/dist/modules/components/dropdown/tag-input-dropdown.component.d.ts:11:15 
    TS2304: Cannot find name 'Ng2Dropdown'.
[at-loader] node_modules/ng2-tag-input/dist/modules/components/dropdown/tag-input-dropdown.component.d.ts:30:28 
    TS2304: Cannot find name 'Ng2Dropdown'.
[at-loader] node_modules/ng2-tag-input/dist/modules/components/dropdown/tag-input-dropdown.component.d.ts:32:28 
    TS2304: Cannot find name 'Ng2MenuItem'.
```

To replicate:
```
git clone https://github.com/preboot/angular2-webpack
cd angular2-webpack
npm i
npm install ng2-tag-input --save # is version 0.8.7 right now
vim src/app/app.module.ts
# Import TagInputModule
npm run server
# go to http://localhost:8080
```

**Comment by Gbuomprisco** [comment]
Possibly related to https://github.com/angular/angular-cli/issues/3674 (I can reproduce with the cli, not with the webpack config in the repo)

**Comment by dland512** [comment]
For anyone running into this issue, it seems to go away if you roll ng2-tag-input back to version 0.8.6 (assuming that's an option for you).

**Comment by gen4sp** [comment]
I am on 0.8.6

**Comment by mtfurlan** [comment]
For the seed I linked, `0.8.6` still has this error.

**Comment by Gbuomprisco** [comment]
just a note: the difference between 0.8.6 and 0.8.7 is just a dev-dependency update, so I am inclined to think that the issue is related to an update in the tooling (copy/paste from another ticket, likely the same issue but with different stacktrace)

**Comment by fahad-nxvt** [comment]
0.8.7 has the same error. Any update on this?

**Comment by gen4sp** [comment]
set "0.8.6." instead of "^0.8.6" fix for me
but waiting for an update

**Comment by fahad-nxvt** [comment]
It helped. Solved the issue for now. Thanks.

**Comment by Gbuomprisco** [comment]
Hi everyone,

I released version 0.8.8. I could not reproduce this issue, but it fixed https://github.com/Gbuomprisco/ng2-tag-input/issues/220, so maybe give it a try? Thanks

**Comment by mtfurlan** [comment]
Seems to work fine now, cheers!
If it pops up again I'll come back with a working example in a docker container that can be replicated on other computers...
